aviary/subsystems/propulsion/hamilton_standard.py

```diff
@@ -689,11 +688,11 @@ def compute(self, inputs, outputs):
                             CP_CLi_table[CL_tab_idx][:cli_len], XPCLI[CL_tab_idx], CPE1X)
                         if (run_flag == 1):
                             ichck = ichck + 1
-                        if (self.options[Settings.VERBOSITY] != Verbosity.DEBUG):
+                        if (verbosity != Verbosity.DEBUG):
```

Comment:
should this be `not Quiet` or `is Debug`?

Reply:
`not Quiet`, `is Debug`, and the line here (`!= Debug`) are all different - are you asking if we should change the behavior, or rewriting it to use identity comparisons instead of equalities? I double-checked and apparently it is not recommended to use equalities for Enums so I'll change it to `not Debug` unless we want to change behavior.

Reply:
right now it is printing for everything other than debug, and not printing if it is debug.
I was wondering if that was intended, or if it was meant to print only for debug, or for everything other than quiet.

Reply:
This isn't my code so I can't speak to the original intent, but I believe this is specifically intended. There are two variants of this output, which is controlled by multiple factors I don't know the origin of - one "normal" printout for everything up to debug level, and a debug specific variant (the `else` part of that comparison).

Reply:
Thanks, I took another look at this, and it looks like the printouts are the exact same, so this could be simplified to an
`if DEBUG or ichck <= 1` with no else